code/modules/vehicles/cargo_train.dm

```diff
@@ -364,7 +364,7 @@
 		move_delay = max(0, (-car_limit * active_engines) + train_length - active_engines)	//limits base overweight so you cant overspeed trains
 		move_delay *= (1 / max(1, active_engines)) * 2 										//overweight penalty (scaled by the number of engines)
 		move_delay += config.run_speed 														//base reference speed
-		move_delay *= 1.1																	//makes cargo trains 10% slower than running when not overweight
```

Comment:
I agree that the cargo train is a bit overpowered, but this is punitive, and would completely destroy its suitability for the intended role.

it shouldn't be able to run easily over corruption, that'll take some more precise coding to do. But even as a stopgap, this is far too severe

Reply:
I've altered these values so far. 3.2 is a sane value. Tested it out myself.